### ISSUE TITLE
[Blocker] It is possible to get into the form fields under the Blocker using the keyboard

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#827]
## Description
User can get to any form field under Blocker using `Tab` key.

## Steps to Reproduce
1. Open [Blocker](https://uui.epam.com/documents?category=components&id=blocker&mode=doc&skin=UUI4_promo) component.
2. Go to Property Explorer (any skin).
3. Press `Tab` key for getting in the fields under "Blocker".

## Actual result
User can get into any field using `Tab` key. User cannot edit the fields under "Blocker".

https://user-images.githubusercontent.com/57751283/202572801-2249d255-8e60-479e-bbb5-bfaed409d811.mp4

## Expected result
User cannot get into the fields under "Blocker" using keyboard, as they cannot do it with the mouse.

## Environment

* UUI version: 4.9.0.
* Browser name: any

**Comment by dzevakov** [comment]
The Block component does not imply control over other components. As a solution to this behavior, a demonstration was added to the documentation with examples for the Blocker component, showing how to block access to control components when the Blocker is enabled where it needed.

**Comment by NatalliaAlieva** [comment]
Released in 4.10.0 version.
